src/process.ml

```diff
@@ -112,7 +112,11 @@ module Fancy = struct
     | x :: rest -> x :: colorize_args rest
 
   let command_line ~prog ~args ~dir ~stdout_to ~stderr_to =
-    let prog = Path.to_string prog in
+    let prog =
+      match dir with
+      | None -> Path.to_string prog
+      | Some from -> Path.reach_for_running prog ~from
```

Comment:
We should do the same as what you did in `run_internal`, i.e. always call `Path.reach_from_running`.

The idea here is that the user can copy and paste the command line in the shell to execute it and debug issues. However `Path.to_string prog` might produce a command that cannot be run, for instance because it contains no `/` and the executable is local.

Reply:
Yeah, I suspected as much. I pushed a commit to use reach_for_running consistently. I also made Path.root the default value for `~from`.